.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: t1m0thyj/unlock-keyring@v1
       - run: |
           go run gotest.tools/gotestsum -- -race -v -count=1 -coverprofile=coverage.out \
-          `go list ./... | grep -Ev 'cmd|docs|examples|pkg/api|tools'`
+          `go list ./... | grep -Ev 'cmd|docs|examples|pkg/api|tools|test|testing|mock'`
```

Comment:
```suggestion
          `go list ./... | grep -Ev 'cmd|docs|examples|pkg/api|tools`
```

no need to include non-existent directories

Reply:
lets return `test|testing` as it also excludes nested directories like

```
internal/db/test
internal/testing
...
```

Reply:
`db test` is actually a command we expose to the end user so it should be included in coverage.

`internal/testing` may be excluded but it's also good to know if those utils are actually used by tests. I'm in favour of keeping.